src/public-modules/Bounties/sagas.js

```diff
@@ -27,7 +27,6 @@ const {
   loadBounties: loadBountiesAction,
   loadBountiesFail,
   loadBountiesSuccess,
-  loadMoreBounties: loadMoreBountiesAction,
```

Comment:
This is overwritten by the function further down - let me know if this is wrong!

Reply:
@codeluggage again - this is renamed. That is the syntax - it's renamed as loadMoreBountiesAction - a var loadMoreBounties isn't grabbed here. This is ok - we should add it back in.  This syntax is similar to import lskdjfd as sdlkfj

Reply:
@villanuevawill Yeah I was getting it wrong when going through with these comments - I actually removed this because it isn't used anywhere in the file. Should it be?

Reply:
If it's not used - then kill it :).